scripts/datasets/machine_translation/wmt2014_ende.sh

```diff
@@ -24,17 +24,21 @@ nlp_preprocess clean_tok_para_corpus --src-lang ${SRC} \
                       --tgt-corpus train.raw.${TGT} \
                       --min-num-words 1 \
                       --max-num-words 100 \
+                      --max-ratio 1.5 \
                       --src-save-path train.tok.${SRC} \
-                      --tgt-save-path train.tok.${TGT}
+                      --tgt-save-path train.tok.${TGT} \
+                      --num-process 32
 
 nlp_preprocess clean_tok_para_corpus --src-lang ${SRC} \
                       --tgt-lang ${TGT} \
                       --src-corpus dev.raw.${SRC} \
                       --tgt-corpus dev.raw.${TGT} \
                       --min-num-words 1 \
                       --max-num-words 100 \
+                      --max-ratio 1.5 \
                       --src-save-path dev.tok.${SRC} \
-                      --tgt-save-path dev.tok.${TGT}
+                      --tgt-save-path dev.tok.${TGT} \
+                      --num-process 32
```

Comment:
Automatically determine the num-process. The user may run the script in other operating system so you should not add the flag.

Reply:
TODO automatically determine the num-process in clean scripts